src/App.js

```diff
@@ -1,9 +1,75 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import VoteButton from './VoteButton';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  list-style: none;
+  font-size: 23px;
+  & li {
+    margin-top: 12px;
+    margin-bottom: 12px;
+  }
+
+  & rank {
+    margin-right: 1rem;
+  }
+`;
+
+const Header = styled.div`
+  font-size: 32px;
+  color: #b3b3ff;
+  padding: 2rem;
+`;
 
 function App() {
+  const [candidates, setCandidates] = useState(null);
```

Comment:
```suggestion
  const [candidates, setCandidates] = useState([]);
```
candidates가 배열을 담는 공간이므로 빈배열[]로 초기화 시켜주는게 의미상 더 좋을거 같기도 하네요